woocommerce/wc-admin
# Add a new note a day after adding the first product

Fixes #5956 

This PR adds a new note with a link to the customizer doc a day after adding the first product.

### Screenshots

![Screen Shot 2021-01-08 at 10 15 42 AM](https://user-images.githubusercontent.com/4723145/104049893-974ab080-519a-11eb-83de-16ad52be8081.jpg)

### Detailed test instructions:

Make sure your store is less than 14 days old. If it has been active for more than 14 days, run the following query.

```update wp_options set option_value = UNIX_TIMESTAMP() where option_name='woocommerce_admin_install_timestamp'```

1. Start with a fresh installation of WooCommerce or delete all published products.
2. Add a new product.
3. Open `wc_posts` and find the product you just added in step #2. Update the `post_date` and `post_date_gmt` to the past date (at least -1 day)
2. Install & activate "WP Crontrol" plugin
3. Navigate to Tools -> Cron Events and run `wc_admin_daily` job
4. Navigate to WooCommerce -> Home and confirm a new note with the title "How to customize your product catalog"
